java/arcs/core/storage/StorageProxy.kt

```diff
@@ -15,9 +15,11 @@ import arcs.core.crdt.CrdtData
 import arcs.core.crdt.CrdtModel
 import arcs.core.crdt.CrdtOperationAtTime
 import arcs.core.crdt.VersionMap
+import arcs.core.storage.StorageProxy.ProxyState.CLOSED
```

Comment:
Not needed - I'm guessing the IDE auto importer was a little too enthusiastic :)